### PR TITLE
Create meywy.json

### PR DESCRIPTION
**Pull request by Meywy**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.  

## Website Link/Preview

https://meywy.netlify.app/

### CODE REVIEW

File: domains/meywy.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    "description": "Portfolio",
+    "repo": "https://github.com/Meywy/meywy.github.io",
+    "owner": {
+        "username": "Meywy",
+        "email": "meywyy@gmail.com",
+        "discord": "923162990419791913"
+    },
+    "record": {
+        "A": [
+            "75.2.60.5"
```
Comment (Stef-00012) [inline domains/meywy.json:11]: Error 404 Not Found

**Review by Stef-DP — DISMISSED** [review]

File: domains/meywy.json (hunk 1 above)
Comment (Meywy) [reply domains/meywy.json:11]: Hey, I dont quite understand, the website https://meywy.netlify.app is accessible normally.

Comment (CuteDog5695) [reply domains/meywy.json:11]: He most probably visited the IP instead of the link

**Review by iostpa — APPROVED** [review]
User listed their site, so it seems good to me

**Pull request merged**